addons/mail/static/src/models/dialog/dialog.js

```diff
@@ -8,6 +8,27 @@ registerModel({
     name: 'Dialog',
     identifyingFields: ['manager', ['attachmentViewer', 'followerSubtypeList']],
     recordMethods: {
+        /**
+         * @param {Element} element
+         * @returns {boolean}
+         */
+        isInContent(element) {
+            return Boolean(this.record && this.record.contains(element));
+        },
```

Comment:
strange naming consistency with those methods

`isInContent` which is "boolean" named, `contains` which is not.

I think the format of `contains` is better if we read it as actual words: "records contains element?", whereas "dialog isInContent element?" doesn't make much sense. `hasElementInContent` maybe?

Also I would probably have "element" in both method name (`containsElement` or something like that), because `contains` is very ambiguous when read alone.

Reply:
`containsElement` and `containsElementInContent`?

Reply:
or if we want to highlight the methods return a boolean, `isContainingElement`, `isContainingElementInContent`?

Reply:
Agree, we tend to read with param name when looking at signature but it's wrong to use param name to read the function. So totally right to rename to `containsElement`